README.md

```diff
@@ -1,5 +1,3 @@
-[![Build Status](https://travis-ci.com/Enterprise-connect/web-ui-admin.svg?branch=v1.1beta)](https://travis-ci.com/Enterprise-connect/web-ui-admin)
```

Comment:
Some lines in this README are removed. E.g. the build badge line#1 Please verify and avoid the removal. @Subhash-vishwakarma @iamkoushikpal-ge